platform/migration/platform-migration/src/main/java/org/codice/ddf/configuration/migration/ExportMigrationManagerImpl.java

```diff
@@ -144,6 +153,13 @@ public void doExport(String productBranding, String productVersion) {
     Validate.notNull(productBranding, "invalid null product branding");
     Validate.notNull(productVersion, "invalid null product version");
     final String ddfHome = System.getProperty("ddf.home");
+    final Path ddfHomePath = Paths.get(ddfHome);
+
+    final Properties systemProperties = AccessUtils.doPrivileged(() -> System.getProperties());
```

Comment:
❗️ We should make the method final if we're doing a privileged check. Otherwise, an extending class has elevated privileges when calling this method.